packages/component-library/src/components/form/mt-link/mt-link.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <component
+    :is="elementType"
+    class="mt-link"
+    :class="linkClasses"
+    :aria-disabled="disabled"
+    :tabindex="disabled ? -1 : 0"
+    v-bind="to ? { ...$attrs, to } : $attrs"
+    @click="onClick"
+  >
+    <slot />
+  </component>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import type { PropType } from "vue";
+
+export default defineComponent({
+  name: "MtLink",
+
+  props: {
+    to: {
+      type: String,
+      required: false,
+      default: undefined,
+    },
+    size: {
+      type: String,
+      required: false,
+      default: "default",
+      validator(value: string) {
+        if (!value.length) {
+          return true;
+        }
+        return ["small", "default", "large"].includes(value);
+      },
+    },
+    variant: {
+      type: String as PropType<"primary" | "secondary" | "critical">,
+      required: false,
+      default: "primary",
+      validator(value: string) {
+        if (!value.length) {
+          return true;
+        }
+        return ["primary", "secondary", "critical"].includes(value);
+      },
+    },
+    disabled: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+    underline: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+    elementType: {
```

Comment:
I would rename that to `as`

Reply:
Thank you, I updated it, please help me confirm